### PR TITLE
Issue 548/distinguish zetkinuser and zetkinperson in mocks

### LINKED ISSUE #548: Distinguish ZetkinPerson and ZetkinUser in mocks
### Description
A user should be able to...

### Requirements
- [ ] Requirement 1

### PR DESCRIPTION
**Pull request by ziggabyte**
## Description
This PR does a bit of house cleaning in the mocks department, especially in regards to making it more obvious if you're referencing a ZetkinPerson or ZetkinUser.

## Changes
* Creates a RosaLuxemburgUser-mock to be used in the few places where a ZetkinUser is needed. 
* Removes a file of unused mocks. 
* Updates mocks to be more streamlined, replacing strings and hardcoded objects with other mocks. 

## Related issues
Resolves #548

### CODE REVIEW

File: playwright/tests/organize/memberships.spec.ts [hunk 1]
```diff
@@ -17,14 +17,15 @@ test.describe('User gets 404 when trying to access org pages', () => {
...
+          id: RosaLuxemburgUser.id,
+          name:
+            RosaLuxemburgUser.first_name + ' ' + RosaLuxemburgUser.last_name,
         },
         role: 'witch',
       },
     ];
 
-    login(RosaLuxemburg, memberships);
+    login(RosaLuxemburgUser, memberships);
```
Comment (richardolsson) [inline playwright/tests/organize/memberships.spec.ts:28]: This is correct!

File: playwright/tests/organize/memberships.spec.ts [hunk 2]
```diff
@@ -40,14 +41,15 @@ test.describe('User gets 404 when trying to access org pages', () => {
       {
         organization: KPD,
         profile: {
-          id: RosaLuxemburg.id,
-          name: RosaLuxemburg.first_name + ' ' + RosaLuxemburg.last_name,
+          id: RosaLuxemburgUser.id,
+          name:
+            RosaLuxemburgUser.first_name + ' ' + RosaLuxemburgUser.last_name,
```
Comment (richardolsson) [inline playwright/tests/organize/memberships.spec.ts:46]: Same issue as above.

**Review by richardolsson — CHANGES_REQUESTED** [review]
I think this looks mostly good. We do have a problem with mock data containing more fields than actual API data, meaning that in theory tests could be falsely positive.

But the types are correct so I don't think it's a huge problem because typescript would catch any misuse of data before the tests even run.

File: playwright/tests/organize/memberships.spec.ts [hunk 3]
```diff
@@ -17,14 +17,15 @@ test.describe('User gets 404 when trying to access org pages', () => {
           title: 'Satanist Monster Org',
         },
         profile: {
-          id: RosaLuxemburg.id,
-          name: RosaLuxemburg.first_name + ' ' + RosaLuxemburg.last_name,
+          id: RosaLuxemburgUser.id,
+          name:
+            RosaLuxemburgUser.first_name + ' ' + RosaLuxemburgUser.last_name,
```
[the remark this answers is not in the document]
Comment (ziggabyte) [reply playwright/tests/organize/memberships.spec.ts:22]: Aha! You caught one i did not see when going throught this! Thank you!

**Review by richardolsson — APPROVED** [review]
Great! 💯

**Pull request merged**